### PR TITLE
Update endpoint-slices.md

### PR DESCRIPTION
**Pull request by skypeter1**
This PR is intended to fix a very minor misspelling on the documentation. 
It is useful to have this when looking for references in a doc page for a particular resource.

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
Please sign CLA
To check `/easycla`

**Comment by utkarsh-singh1** [comment]
Hi @skypeter1, please follow these guides to sign easycla,


You can find about EasyCLA here - https://docs.linuxfoundation.org/lfx/easycla

and the steps to sign EasyCLA are here - https://docs.linuxfoundation.org/v2/easycla/contributors/individual-contributor

You can start your signing EasyCLA from here - https://api.easycla.lfx.linuxfoundation.org/v2/repository-provider/github/sign/18706487/51478266/38056/#/?version=2

File: content/en/docs/concepts/services-networking/endpoint-slices.md [hunk 1]
```diff
@@ -235,7 +235,7 @@ at different times.
 {{< note >}}
 Clients of the EndpointSlice API must iterate through all the existing EndpointSlices
 associated to a Service and build a complete list of unique network endpoints. It is
-important to mention that endpoints may be duplicated in different EndointSlices.
+important to mention that endpoints may be duplicated in different EndpointSlices.
```
Comment (dipesh-rawat) [inline content/en/docs/concepts/services-networking/endpoint-slices.md:238]: This small fix looks good to me.

Deploy Preview: https://deploy-preview-39745--kubernetes-io-main-staging.netlify.app/docs/concepts/services-networking/endpoint-slices/#duplicate-endpoints

Comment (bradtopol) [inline content/en/docs/concepts/services-networking/endpoint-slices.md:238]: Great job!

**Pull request merged**